applications/ecallmgr/src/ecallmgr_config.erl

```diff
@@ -30,7 +30,7 @@
 -include_lib("kazoo/include/kz_databases.hrl").
 
 -spec flush() -> 'ok'.
--spec flush(kz_json:key()) -> 'ok' | {'error', any()}.
+-spec flush(kz_json:keys()) -> 'ok' | {'error', any()}.
```

Comment:
I agree that the types could be improved, naming-wise, to increase clarity. I think the original decision was made because having `-type foo() :: whatever()` made having many `foo` have type `-type foos() :: [foo()]`.

That said, I agree with the proposal of making `key() :: [json_string()] | json_string().`